ui/components/TracePage/TracePage.tsx

```diff
@@ -149,7 +150,7 @@ const TreeProvider = ({ traceId, children }: { traceId: string; children: ReactN
         if (offset >= contentLength) return;
 
         const initialOffset = 1e6;
-        const subsequentOffset = 1e7;
+        const subsequentOffset = 1e9;
```

Comment:
This looks bad. I'd guess we should be sticking to 1e6 at all times now that blocks are lazily loaded.